tests/test_annotations.rs

```diff
@@ -27,47 +27,47 @@ test!(test_annotations_scopes_1, {
         .expect("Could not annotate AST");
 
     // Toplevel
-    assert_eq!(ast["BINJS:Scope"], object!{
+    assert_eq!(ast["BINJS:Scope"], json!({
         "type" => "BINJS:Scope",
```

Comment:
Does serde understand `=>` in `json!`?

Reply:
Hmm good catch. I don't think it does, but not sure how these tests pass either...

Reply:
Okay, it's definitely not supported. I'm confused. Does this test even run?

Reply:
Oh right, it's all commented out...

Reply:
@Yoric Fixed but TBH it feels weird to maintain large commented out block of code... Can we either uncomment it out (if it works) or delete and rely on Git history should we ever want to restore it?

Reply:
You're right, that is weird.

Could you quickly check whether it works? Otherwise, let's get rid of it.

Reply:
It doesn't, mostly broken by the fact that `parse_str` now returns statically typed objects and not JSON.

Reply:
But even before that, I think the JSON keys used weren't valid for a while now.

Reply:
Removed test in a separate commit and rebased branch on top of it.